### ISSUE TITLE
One of the default glb avatars is malfunctioning (500 internal server error)

### ISSUE THREAD

**Issue by utkarsh-promact** [croquet/microverse#70]
Avatar url => "https://models.readyplayer.me/62fd66f93e172e005443e7cc.glb" is broken from ready player me's end. This needs to be replaced or removed from the code. 

This code is on line 38 to 44 in src/settingsMenu.js

**Comment by yoshikiohshima** [comment]
Thanks for reporting. Interestingly only one of 8 does not load. Let us investigate what is going on.

**Comment by utkarsh-promact** [comment]
Is this issue resolved? or if possible when can  it be resolved as we are using croquet in deployed applications and this error is persistent

**Comment by e46mpwr** [comment]
Yes, this issue has been fixed.  The GLB is available from the reported link.

**Comment by yoshikiohshima** [comment]
Oh, ok. Yes, it does work. Did you have to contact them? In any case, I am ready to make a change to host those models on croquet.io and load them from there, instead of the Ready Player Me site.

**Comment by utkarsh-promact** [comment]
Please do update here once hosted from croquet.io , would be really helpful

**Comment by e46mpwr** [comment]
> Oh, ok. Yes, it does work. Did you have to contact them? In any case, I am ready to make a change to host those models on croquet.io and load them from there, instead of the Ready Player Me site.

Yes, I contacted RPM.  This was their response.  "The Avatar seems to be working now. It is possible that our development team was deploying some updates to our service and the URL was temporarily unavailable."
